tests/rptest/tests/services_self_test.py

```diff
@@ -367,9 +367,12 @@ def test_kubectl_tool(self):
                 pass
 
 
+from ducktape.mark.resource import cluster as dt_cluster
+
+
 class KubectlLocalOnlyTest(Test):
 
-    # @cluster(num_nodes=0)
+    @dt_cluster(num_nodes=0)
```

Comment:
Note that without being explicit (i.e. `@cluster(num_nodes=...`), each test case by default asks for [ALL the available nodes](https://ducktape.readthedocs.io/en/latest/new_tests.html?highlight=ducktape.mark.resource.cluster#ducktape.mark.resource.cluster)  in session.  Two additional conditions in combination causes test failure by `"Test requested X nodes, but used 0"`:
* This test should not, and does not actually use any nodes.
* ducktape is sometimes invoked with `--fail-bad-cluster-utilization`, depending on the context.  `cdt_cloud.sh` does NOT use `--fail-bad-cluster-utilization`, but other entrypoints into our DT tests often DO.

We cannot use our custom `@cluster` defined within our `rptest` framework because it **requires** the test to have a `redpanda` attribute - which we do not want (we don't want to spin up any services nor utilize any nodes).

So we fallback to using the regular `@cluster` decorator, to specify that we need 0 nodes for this test.